test/functional/api/cas/core.py

```diff
@@ -49,11 +50,24 @@ def get_core_statistics(self,
                             io_class_id: int = None,
                             stat_filter: List[StatsFilter] = None,
                             percentage_val: bool = False):
+        stats = get_statistics(self.cache_id, self.core_id, io_class_id,
+                               stat_filter, percentage_val)
+        if io_class_id is not None:
+            return IoClassStats(stats)
+        return CoreStats(stats)
```

Comment:
Ditto. Please provide separate function for core io class stats.

Reply:
done